es79x/src/main/scala/tech/beshu/ror/es/request/context/types/ResizeEsRequestContext.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.es.request.context.types
+
+import cats.data.NonEmptyList
+import cats.implicits._
+import org.elasticsearch.action.admin.indices.shrink.ResizeRequest
+import org.elasticsearch.threadpool.ThreadPool
+import tech.beshu.ror.accesscontrol.AccessControlStaticContext
+import tech.beshu.ror.accesscontrol.domain.IndexName
+import tech.beshu.ror.es.RorClusterService
+import tech.beshu.ror.es.request.AclAwareRequestFilter.EsContext
+import tech.beshu.ror.es.request.context.ModificationResult
+import tech.beshu.ror.es.request.context.ModificationResult.{Modified, ShouldBeInterrupted}
+
+class ResizeEsRequestContext(actionRequest: ResizeRequest,
+                             esContext: EsContext,
+                             aclContext: AccessControlStaticContext,
+                             clusterService: RorClusterService,
+                             override val threadPool: ThreadPool)
+  extends BaseIndicesEsRequestContext[ResizeRequest](actionRequest, esContext, aclContext, clusterService, threadPool) {
+
+  override protected def indicesFrom(request: ResizeRequest): Set[IndexName] = {
+    (request.getSourceIndex :: request.getTargetIndexRequest.index() :: Nil)
```

Comment:
I see that target index request can have aliases. We should also add them here. ResizeRequest with target index with at least one forbidden alias should be rejected.